specification/schemas/PaymentIdentityRelationship.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "object",
+  "required": [
+    "data"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "data": {
+      "type": "object",
+      "required": [
+        "type"
```

Comment:
`id` should be required and you should add a `related` link.

Reply:
Payment Identity does not have a endpoint yet for the related link

Reply:
There should be an endpoint for it in the current spec.